commercionetwork/chain.json

```diff
@@ -21,19 +21,19 @@
   },
   "codebase": {
     "git_repo": "https://github.com/commercionetwork/commercionetwork",
-    "recommended_version": "v4.0.0",
+    "recommended_version": "v5.0.0",
     "compatible_versions": [
-      "v4.0.0"
+      "v5.0.0"
     ],
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/commercionetwork/chains/master/commercio-3/genesis.json"
     },
     "versions": [
       {
-        "name": "v4.0.0",
-        "recommended_version": "v4.0.0",
+        "name": "v5.0.0",
```

Comment:
instead of writing over v4, could you please keep v4 in this array and add a new object below for v5?

Reply:
Updated versions array. Thank you